docx/from/docxtotei.xsl

```diff
@@ -1,713 +1,750 @@
-	  of the conversion, cleaning TEI elements created in the
-	  first ise."</li>
-	  
-	  <li>inSectionGroup: defines a template that works on a
-	  group of consecutive elements (w:p or w:tbl elements) that
-	  form a section (a normal section, not to be confused with
-	  w:sectPr).</li>
-	  
-	  <li>paragraph: defines that the template
-	  works on an individual element (usually
-	  starting with a w:p element).  </li>
-	  
-	  <li>iden: simply copies the content</li>
-	</ul>
-	
-      </desc>
-   </doc>
-   <xsl:template match="/">
-     <!-- Do an initial normalization and store everything in $pass0 -->
-      <xsl:if test="not(doc-available($relsFile))">
-	<xsl:message terminate="yes">The file <xsl:value-of
-	select="$relsFile"/> cannot be read</xsl:message>
-      </xsl:if>
-      <xsl:if test="not(doc-available($styleDoc))">
-	<xsl:message terminate="yes">The file <xsl:value-of
-	select="$styleDoc"/> cannot be read</xsl:message>
-      </xsl:if>
-     <xsl:variable name="pass0">
-       <xsl:apply-templates mode="pass0"/>
-     </xsl:variable>
-     
-     <!-- Do the main transformation and store everything in the variable pass1 -->
-     <xsl:variable name="pass1">
-       <xsl:for-each select="$pass0">
-	 <xsl:apply-templates/>
-       </xsl:for-each>
-     </xsl:variable>		  
-     
-
-     <!--
-	 <xsl:result-document href="/tmp/foo.xml">
-	 <xsl:copy-of select="$pass1"/>
-	 </xsl:result-document>
-     -->
+  xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+  xmlns:prop="http://schemas.openxmlformats.org/officeDocument/2006/custom-properties"
+  xmlns:a="http://schemas.openxmlformats.org/drawingml/2006/main"
+  xmlns:cp="http://schemas.openxmlformats.org/package/2006/metadata/core-properties"
+  xmlns:dc="http://purl.org/dc/elements/1.1/"
+  xmlns:dcterms="http://purl.org/dc/terms/"
+  xmlns:dcmitype="http://purl.org/dc/dcmitype/"
+  xmlns:iso="http://www.iso.org/ns/1.0"
+  xmlns:m="http://schemas.openxmlformats.org/officeDocument/2006/math"
+  xmlns:mml="http://www.w3.org/1998/Math/MathML"
+  xmlns:mo="http://schemas.microsoft.com/office/mac/office/2008/main"
+  xmlns:mv="urn:schemas-microsoft-com:mac:vml"
+  xmlns:o="urn:schemas-microsoft-com:office:office"
+  xmlns:pic="http://schemas.openxmlformats.org/drawingml/2006/picture"
+  xmlns:r="http://schemas.openxmlformats.org/officeDocument/2006/relationships"
+  xmlns:rel="http://schemas.openxmlformats.org/package/2006/relationships"
+  xmlns:tbx="http://www.lisa.org/TBX-Specification.33.0.html"
+  xmlns:html="http://www.w3.org/1999/xhtml"
+  xmlns:tei="http://www.tei-c.org/ns/1.0"
+  xmlns:teidocx="http://www.tei-c.org/ns/teidocx/1.0"
+  xmlns:v="urn:schemas-microsoft-com:vml"
+  xmlns:ve="http://schemas.openxmlformats.org/markup-compatibility/2006"
+  xmlns:w10="urn:schemas-microsoft-com:office:word"
+  xmlns:w="http://schemas.openxmlformats.org/wordprocessingml/2006/main"
+  xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
```

Comment:
Really! That seems worth doing.

On Wed, Jul 27, 2016 at 12:36 PM, Erik Hetzner <notifications@github.com>
wrote:

> In docx/from/docxtotei.xsl
> <https://github.com/Tahi-project/Stylesheets/pull/8#discussion_r72474816>:
>
> > +  xmlns:mml="http://www.w3.org/1998/Math/MathML"
> > +  xmlns:mo="http://schemas.microsoft.com/office/mac/office/2008/main"
> > +  xmlns:mv="urn:schemas-microsoft-com:mac:vml"
> > +  xmlns:o="urn:schemas-microsoft-com:office:office"
> > +  xmlns:pic="http://schemas.openxmlformats.org/drawingml/2006/picture"
> > +  xmlns:r="http://schemas.openxmlformats.org/officeDocument/2006/relationships"
> > +  xmlns:rel="http://schemas.openxmlformats.org/package/2006/relationships"
> > +  xmlns:tbx="http://www.lisa.org/TBX-Specification.33.0.html"
> > +  xmlns:html="http://www.w3.org/1999/xhtml"
> > +  xmlns:tei="http://www.tei-c.org/ns/1.0"
> > +  xmlns:teidocx="http://www.tei-c.org/ns/teidocx/1.0"
> > +  xmlns:v="urn:schemas-microsoft-com:vml"
> > +  xmlns:ve="http://schemas.openxmlformats.org/markup-compatibility/2006"
> > +  xmlns:w10="urn:schemas-microsoft-com:office:word"
> > +  xmlns:w="http://schemas.openxmlformats.org/wordprocessingml/2006/main"
> > +  xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
>
> You can just change mc:Choice to ve:Choice. In a proper XML parser the
> prefix doesn't matter, only the URI that it maps to.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/Tahi-project/Stylesheets/pull/8/files/8feacc95c00616da21e45ddb6c282803e66eb995#r72474816>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AB8t1KrBHNRa0xW8LmO-dElLKDS4fL8Iks5qZ4j_gaJpZM4JMeq0>
> .
>